### DISCUSSION TITLE
[Docs] 能不能查看低版本的文档？

### DISCUSSION THREAD

**Discussion by h-ecstasy** [element-plus/element-plus#8461] [Q&A] [answered]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

项目用的是低版本的UI库，版本升级你们的文档也随之升级了，低版本文档也找不到。。。

### Additional comments

_(empty)_

**Comment by sxzz** [comment]
Refer to #7230, and download by commit history https://github.com/element-plus/element-plus/commits/gh-pages

**Reply by kimmy-wang** [reply]
官网文档已经支持国际化了，但是源代码中docs目录并没有国际化相关的代码

**Reply by KimYangOfCat** [reply]
参考 #8444

**Comment by KimYangOfCat** [comment, marked as answer]
也可以尝试从 releases 中下载对应版本的源码：

1. 下载 release 中对应所使用版本的源码，
2. 执行 nvm install 安装对应版本的 nodejs
3. 执行 pnpm install 安装依赖，
4. 执行 pnpm run docs:dev 启动文档的本地开发环境

相比前文所提到的方法，优势是不用自己手动去启动一个 http sever 环境，但由于 #8444 中提到的限制，这样只能在本地启动英文文档，无法访问中文文档的数据

**Reply by h-ecstasy** [reply]
只能看英文文档，没有中文文档
